plugins/flytekit-spark/flytekitplugins/spark/task.py

```diff
@@ -141,6 +142,11 @@ def __init__(
             if container_image.base_image is None:
                 img = f"cr.flyte.org/flyteorg/flytekit:spark-{DefaultImages.get_version_suffix()}"
                 self._container_image = dataclasses.replace(container_image, base_image=img)
+                builder = ImageBuildEngine._get_builder(container_image.builder)
+                if builder == "default":
```

Comment:
is this right? `_get_builder` returns `ImageSpecBuilder`.

Reply:
no, sorry, I was testing something